### DISCUSSION TITLE
All mails in queue

### DISCUSSION THREAD

**Discussion by sukallol** [postalserver/postal#1590] [Help with using Postal]
Hi

can anyone help me regarding this.
![err](https://user-images.githubusercontent.com/90283018/132424056-2fb8d7dc-03b3-4f41-845e-d7b7228bb4c5.jpg)

**Comment by willpower232** [comment]
What is the output of `postal status`?

**Reply by sukallol** [reply]
![1](https://user-images.githubusercontent.com/90283018/132474041-bf0bda9c-af29-4a1d-8676-6efe539c05c7.jpg)

**Reply by sukallol** [reply]
![1](https://user-images.githubusercontent.com/90283018/132474218-040306c4-09aa-47f1-8809-2e70881d7a87.jpg)

**Reply by willpower232** [reply]
Now that Postal is running, you may be able to send emails again. It is possible that the server was rebooted and did not start in a good order, if that is the case, have a look at #1562

**Reply by sukallol** [reply]
still the same problem, when i click on send it keeps on processing

**Reply by sukallol** [reply]
in logs i am getting postal vhost error p

**Reply by sukallol** [reply]
root@smtp:~# postal logs worker
Attaching to postal_worker_1
worker_1    | [9] [2021-09-07T18:04:40.987] INFO -- : Worker running with 4 threads
worker_1    | /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `raise': Connection-level error: NOT_ALLOWED - vhost postal not found (Bunny::NotAllowedError)
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `open_connection'
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:320:in `start'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:23:in `create_connection'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:28:in `create_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:195:in `job_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:17:in `work'
worker_1    |   from script/worker.rb:3:in `<main>'
worker_1    | [9] [2021-09-07T19:40:40.065] INFO -- : Worker running with 4 threads
worker_1    | /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `raise': Connection-level error: NOT_ALLOWED - vhost postal not found (Bunny::NotAllowedError)
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `open_connection'
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:320:in `start'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:23:in `create_connection'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:28:in `create_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:195:in `job_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:17:in `work'
worker_1    |   from script/worker.rb:3:in `<main>'
worker_1    | [9] [2021-09-07T19:41:12.192] INFO -- : Worker running with 4 threads
worker_1    | /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `raise': Connection-level error: NOT_ALLOWED - vhost postal not found (Bunny::NotAllowedError)
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `open_connection'
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:320:in `start'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:23:in `create_connection'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:28:in `create_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:195:in `job_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:17:in `work'
worker_1    |   from script/worker.rb:3:in `<main>'
worker_1    | [9] [2021-09-08T08:26:21.696] INFO -- : Worker running with 4 threads
worker_1    | /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `raise': Connection-level error: NOT_ALLOWED - vhost postal not found (Bunny::NotAllowedError)
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `open_connection'
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:320:in `start'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:23:in `create_connection'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:28:in `create_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:195:in `job_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:17:in `work'
worker_1    |   from script/worker.rb:3:in `<main>'
worker_1    | [9] [2021-09-08T08:29:49.496] INFO -- : Worker running with 4 threads
worker_1    | /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `raise': Connection-level error: NOT_ALLOWED - vhost postal not found (Bunny::NotAllowedError)
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:1275:in `open_connection'
worker_1    |   from /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/session.rb:320:in `start'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:23:in `create_connection'
worker_1    |   from /opt/postal/app/lib/postal/rabbit_mq.rb:28:in `create_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:195:in `job_channel'
worker_1    |   from /opt/postal/app/lib/postal/worker.rb:17:in `work'
worker_1    |   from script/worker.rb:3:in `<main>'

**Reply by willpower232** [reply]
> vhost postal not found

Make sure you have set up Rabbit MQ properly https://docs.postalserver.io/install/prerequisites

**Comment by sukallol** [comment]
wget https://postal.atech.media/packages/stable/latest.tgz

in the old installation procedure the above mention command doesnot work as the url have been removed do we have any alternatives for this?

**Reply by willpower232** [reply]
see #1521, it looks like you're already using the new way of doing things.
